nipype/pipeline/plugins/base.py

```diff
@@ -262,7 +262,6 @@ def run(self, graph, config, updatehash=False):
             if toappend:
                 self.pending_tasks.extend(toappend)
             num_jobs = len(self.pending_tasks)
-            logger.debug('Number of pending tasks: %d' % num_jobs)
```

Comment:
This trace appeared too often, maybe it's ok to remove it. But I ask @satra first.